### DISCUSSION TITLE
Add `accept` attribute to `type="file"` inputs to limit types available for upload

### DISCUSSION THREAD

**Discussion by alies-dev** [laravel/nova-issues#5132] [Q&A] [answered]
Docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept

**Comment by crynobone** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/172966/204941995-fe382658-5e80-4b51-aae7-8a0c33cc75e3.png)

It is in the documentation. https://nova.laravel.com/docs/4.0/resources/file-fields.html#customizing-accepted-file-types

**Reply by alies-dev** [reply]
Ooops, I've missed it somehow. Thanks a lot!
